frontend/src/pages/interview.js

```diff
@@ -302,14 +310,15 @@ class Interview extends Component<Props> {
               Explain your reasoning for your score.{' '}
```

Comment:
these single quotes are messing up highlighting again

Reply:
try vscode 😛

They shouldn't be there anyways. just removed them